### ISSUE TITLE
Increase timeout on theme activation snackbar

### ISSUE THREAD

**Issue by dsas** [Automattic/wp-calypso#97091]
### Quick summary

When you activate a theme a message briefly appears to tell you it's successful and to give you a chance to go and look at your site with its shiny new theme.

![Image](https://github.com/user-attachments/assets/e7dc6e57-4207-484a-a9f6-d0f882eba3a3)

This message disappears after ten seconds and you can't bring it back. It's arguably an accessibility failure ([WCAG 2.2 Guideline 2.1: Enough time](https://www.w3.org/WAI/WCAG22/Understanding/enough-time.html)) though there are other ways to view your site, and to realise what the active theme is.

The [timeout is configurable](https://github.com/Automattic/wp-calypso/blob/a594f9310b57b5bdc405a36f1734f7bdbc6fd54e/client/state/themes/actions/activate-theme.js#L71) and we did note when implementing it that [we should review this based upon feedback](https://github.com/Automattic/wp-calypso/pull/94927#discussion_r1782215462). 

Design's initial guidance (p1725899556548679/1725851524.314329-slack-CRWCHQGUB) was:
> snackbars are best for brief and non-critical notifications, like in this case. If they don’t contain a next step, they should auto-close after a few seconds. Otherwise we should keep them visible so users can click if they want

It's a grey area here because there's an optional next step. I think we should probably remove the timeout, but at the least, increase it to 20 seconds.

This was initially prompted by some feedback from @lizswafford : paYJgx-5CF-p2#comment-5604

### Steps to reproduce

1. Go to wordpress.com/themes
2. Choose a theme
3. If present, click "Select a site for this theme" and choose a site
4. Click "Activate this design"
5. Click "Activate [theme name]" on the modal
6. Notice the snackbar appear in the top right

### What you expected to happen

The snackbar to stay around until I dismissed it

### What actually happened

The snackbar disappeared, to view what my site looks like in the theme I selected there's at least two clicks. Most quickly by hovering over the site's name in the masterbar and then clicking "view site"

### Impact

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### If the above answer is "Yes...", outline the workaround.

Assuming you noticed and read the text in ten seconds, you can view the site in many ways.

**Comment by okmttdhr** [comment]
I have increased the timeout in https://github.com/Automattic/wp-calypso/pull/97772. Also, added the track event to revisit the result. 
I believe we can close this issue for now.
